<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameEngine</class>
 <widget class="QMainWindow" name="GameEngine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>884</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Virus Fighter Game</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>601</width>
      <height>811</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>431</width>
      <height>781</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>126</width>
      <height>463</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QPushButton" name="Esc">
               <property name="text">
                <string>Quit</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>STZhongsong</family>
              </font>
             </property>
             <property name="text">
              <string>      Health</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="Health">
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>STZhongsong</family>
          </font>
         </property>
         <property name="text">
          <string>       Score</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLCDNumber" name="Score"/>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>STZhongsong</family>
        </font>
       </property>
       <property name="text">
        <string>        Level</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="Level"/>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>STZhongsong</family>
        </font>
       </property>
       <property name="text">
        <string>     Time left</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="Time_left"/>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>550</y>
      <width>151</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>490</y>
      <width>124</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>STZhongsong</family>
     </font>
    </property>
    <property name="text">
     <string>Status Message</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
